### PR TITLE
add `docker-it` by @whisklabs

### PR DESCRIPTION
**Pull request by agebhar1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 * [bocker](https://github.com/p8952/bocker) (1) - Docker implemented in 100 lines of bash by [p8952](https://github.com/p8952)
 * [construi](https://github.com/lstephen/construi) - Run your builds inside a Docker defined environment by [@lstephen](https://github.com/lstephen)
 * [Docker Client for JVM](https://github.com/gesellix/docker-client) - A Docker remote api client library for the JVM, written in Groovy by [@gesellix][gesellix]
+* [docker-it](https://github.com/whisklabs/docker-it-scala) - Docker integration testing kit with Scala by [@whisklabs](https://github.com/whisklabs)
```
Comment (vegasbrianc) [inline README.md]: Hi @agebhar1 I noticed the name is slightly different that the project. Could you please rename the project to docker-it-scale according to the GitHub project? Thanks!

**Comment by vegasbrianc** [comment]
Hi @agebhar1 Looks like a great tool. Once the name is updated according to my review we can merge this. Thanks!

File: README.md (hunk 1 above)
Comment (agebhar1) [inline README.md]: Hi @vegasbrianc sure will rename it. My fault ;)

**Comment by agebhar1** [comment]
@vegasbrianc - fixed it :)

**Comment by veggiemonk** [comment]
@agebhar1 well done!

If you are interested, would you like to become a maintainer?

**Comment by agebhar1** [comment]
thx @veggiemonk - and yes I'm interested :)

**Comment by veggiemonk** [comment]
@agebhar1 invite sent 👍 welcome.

few things you have to know: 
* we don't push directly to master, we create a pull request.
* how to tell if a project should be on the list, read [CONTRIBUTING.md](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) 
* to ask a contributor to make a change, just copy paste this message [here](https://github.com/veggiemonk/awesome-docker/pull/289#issuecomment-285608004) and changes a few things like names and stuff. The main idea is to help people get to quality.
* If something seems weird, if you don't understand what a project does or the documentation is poor, don't hesitate to ask for more (see previous point).

Anyway welcome to the team, if you have any questions, please ask (use the chat, or twitter or whatever feels more confortable for you)

**Pull request merged**